### PR TITLE
Increase max RPM on bebop 2 and fix comment

### PR DESCRIPTION
**Pull request by jberaud**
Add a Util function on AP_HAL_Linux to detect hardware reading /proc/cpuinfo. This is valid on arm32 platforms but not on arm64, x86 or amd64 platforms.
Fix comment on GPIO_Bebop inherited from minlure.

### CODE REVIEW

**Comment by rmackay9** [comment]
@lucasdemarchi, @tridge, are you guys ok with the "get_hw" call?

File: libraries/AP_HAL_Linux/Util.h [hunk 1]
```diff
@@ -9,6 +9,12 @@
 #include "ToneAlarmDriver.h"
 #include "Semaphores.h"
 
+enum hw_type {
+    HARDWARE_TYPE_BEBOP = 0,
+    HARDWARE_TYPE_BEBOP2,
+    NUM_HARDWARES,
+};
```
Comment (lucasdemarchi) [inline libraries/AP_HAL_Linux/Util.h]: I think you are mixing a generic implementation with a bebop-only implementation. If it's bebop-only, then you shouldn't export hw_type in the header, and you don't need _hw_names in the class. If it's generic, you should bring the get_rpi_version() method to the Util class and merge them.

Comment (jberaud) [inline libraries/AP_HAL_Linux/Util.h]: Ok, I had missed this one. This implementation is generic and I will bring get_rpi_version to the Util class instead. Thanks.

**Comment by jberaud** [comment]
@lucasdemarchi updated accordingly

**Comment by tridge** [comment]
I think its OK for now, but we really need an /etc/ardupilot.conf soon!

**Comment by tridge** [comment]
(Lucas, push when you are happy with it)

**Comment by rmackay9** [comment]
@tridge, @lucasdemarchi, off topic but on one of the upcoming dev calls I'd like to discuss setting some default locations on linux boards for other things as well like log files, video files, pictures, etc.

**Comment by lucasdemarchi** [comment]
@rmackay9 @tridge agreed

**Comment by lucasdemarchi** [comment]
Applied.

**Pull request closed without merging**